ivy/functional/frontends/paddle/tensor/creation.py

```diff
@@ -76,3 +76,22 @@ def eye(num_rows, num_columns=None, dtype=None, name=None):
 @to_ivy_arrays_and_back
 def empty_like(x, dtype=None, name=None):
     return ivy.empty_like(x, dtype=dtype)
+
+
+@with_unsupported_dtypes(
+    {
+        "2.4.2 and below": (
+            "uint8",
+            "int8",
+            "int16",
+            "float16",
+            "complex64",
+            "complex128",
+            "bool",
+        )
+    },
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def tril(x, name=None):

```

Comment:
Hi, thanks for contributing! Could you check this with https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/tril_en.html#tril please?